www/md_base/src/styles/styles.less

```diff
@@ -4,3 +4,7 @@
 [ng-cloak] {
   display: none !important;
 }
+
+a {
+  text-decoration: none;
```

Comment:
hmmm... many websites like Google has removed underline for all links these days. So I tried removing them and found the appearance is good. Maybe we can display underline only when `hover`?